Week3/database.js

```diff
@@ -0,0 +1,106 @@
+const mysql = require('mysql');
+const util = require('util');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'todo_data'
+});
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+const getAll = async function (id) {
+  let sql = `select * from users a join toDosList b on a.Id = b.UserId join todoItems c on b.listId = c.listId where a.Id = ?`
+  try {
+    return await execQuery(sql, [id]);
+  } catch (error) {
+    connection.end();
+    return error;
+  }
+};
+const createUser = async function (id, firstName, lastName) {
+  let sql = `insert into Users (id,firstName,lastName) values (?,?,?)`
+  try {
+    return await execQuery(sql, [id, firstName, lastName]);
+  } catch (error) {
+    connection.end();
+    return error;
+  }
+};
+const createList = async function (UserId, category, prioritize) {
```

Comment:
Just for consistency, I would change the name of this ```UserId``` to ```userId```. Tools like lint can help you with that: https://eslint.org/docs/rules/camelcase =)